src/Framework/Constraint/Exception.php

```diff
@@ -54,7 +54,8 @@ protected function failureDescription($other)
     {
         if ($other !== null) {
             $message = '';
-            if ($other instanceof Exception) {
+            $throwable = 'Throwable';
```

Comment:
You are right. I was told that '$other instanceof Throwable' will cause fatal in PHP5 and didn't check this.